resources/lib/ytchannels.py

```diff
@@ -70,8 +80,21 @@ def ytchannels_main():
 			rem_uri = build_url({'mode': 'rem_channel', 'channel_id': '%s'%str(channels[i][1])})
 			add_uri = build_url({'mode': 'add_folder'})
 			addch_uri = build_url({'mode': 'add_channel', 'foldername': 'Other'})
-			li.addContextMenuItems([(local_string(30003), 'RunPlugin(%s)'%rem_uri),('Add folder', 'RunPlugin(%s)'%add_uri),
-									(local_string(30002), 'RunPlugin(%s)'%addch_uri)])
+			move_down_uri = build_url({'mode': 'move_down', 'id': '%s'%channels[i][4]})
+			move_up_uri = build_url({'mode': 'move_up', 'id': '%s'%channels[i][4]})
+			items = []
+			items.append((local_string(30003), 'RunPlugin(%s)'%rem_uri))
+			items.append(('Add folder', 'RunPlugin(%s)'%add_uri))
+			items.append((local_string(30002), 'RunPlugin(%s)'%addch_uri))
+			if channels[i][3] == 1:
```

Comment:
This shows the "move down" option if you only have 1 item in the list, which you can then do and the item ends up wierdly the second row.

Reply:
fixed